<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookDialog</class>
 <widget class="QDialog" name="bookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="from_citylab">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出发</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>---------------------------&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="to_citylab">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目的</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_codelab">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CA34次航班</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="start_timelab">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2022-12-01 08:15:00出发</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="eco_radioBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>经济舱/1000元</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="busy_radioBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商务舱/2000元</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="best_radioBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>头等舱/3000元</string>
   </property>
  </widget>
  <widget class="QLabel" name="textlab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>671</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>===============================选择乘客==================================</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="passgTabWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>220</y>
     <width>611</width>
     <height>181</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>选择</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电话</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>身份证号码</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="bookpushBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>bookDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
